lib/fog/ovirt/requests/compute/v4/list_virtual_machines.rb

```diff
@@ -3,16 +3,18 @@ module Compute
     class Ovirt
       class V4
         class Real
+          # rubocop:disable Metrics/AbcSize
```

Comment:
Instead of disabling this metric, could we just refactor all `filters` assignments into a private method?